src/dotnet-svcutil/lib/src/dotnet-svcutil-lib.csproj

```diff
@@ -65,7 +65,13 @@
     <PackageReference Include="System.Runtime.Loader" Version="4.3.0" />
     <PackageReference Include="System.Reflection.DispatchProxy" Version="4.4.0" />
   </ItemGroup>
-
+  
+  <Target Name="PackAdditonalDlls" AfterTargets="build">
```

Comment:
Looks good, do we also need to pack the 8.0 version?

Reply:
updated.